rmi/clientsrc/client.java

```diff
@@ -537,7 +537,7 @@ else if (arguments.size()==2)  //command was "help <commandname>"
             try{
               Id = obj.getInt(arguments.elementAt(1));
               int customer = obj.getInt(arguments.elementAt(2));
-              Vector flightNumbers = new Vector();
+              Vector flightNumbers = new Vector<>();

```

Comment:
idk why they though using vectors was a good idea if they don't even know how to use them